src/Domain/FunctionsOfTime/QuaternionHelpers.hpp

```diff
@@ -10,7 +10,13 @@
 #include <boost/math/quaternion.hpp>
 
 #include "DataStructures/DataVector.hpp"
-#include "Utilities/Gsl.hpp"
+
+/// \cond
+namespace gsl {
+template <class T>
+class not_null;
+}
```

Comment:
Please run clang-format over this file, or add a closing `}  // gsl` comment here yourself. Clang-tidy will complain otherwise :)